www/base/src/styles/animations.less

```diff
@@ -38,6 +38,19 @@ tl;dr is:
   }
 }
 
+/* animation for  */
+.anim-fade {
+    -webkit-transition: .5s linear all;
```

Comment:
Only `-webkit` ? No `-moz`, `-o` ?

In fact I think `-webkit-transition` is not needed either. Currently any modern browser that supports `transition` seems has already removes the prefix.